modules/stanford_profile_styles/lib/scss/component/paragraph/card/_card.scss

```diff
@@ -14,8 +14,15 @@
       @include type-e;
     }
 
-    .su-font-splash {
-      font-size: 1.4em;
+    // Overwrite conflicting decanter styles for splash font card headline
+    >div {
+      &:not(.su-card__button) {
+        &.su-font-splash {
```

Comment:
this kinda annoys me with our current codeclimate settings.

Reply:
What issues does this cause related to our codeclimate settings? Just curious.